src/bin/rekordcrate-anlz.rs

```diff
@@ -6,14 +6,15 @@
 //
 // SPDX-License-Identifier: MPL-2.0
 
+use binrw::BinRead;
 use rekordcrate::anlz::ANLZ;
 
 fn main() {
     let path = std::env::args().nth(1).expect("no path given");
-    let data = std::fs::read(&path).expect("failed to read file");
-    let (input, anlz) = ANLZ::parse(&data).expect("failed to parse header");
+    let mut reader = std::fs::File::open(&path).expect("failed to open file");
+    let anlz = ANLZ::read(&mut reader).expect("failed to parse setting file");
     println!("File Header: {:#?}", anlz);
-    for (i, section) in anlz.sections(input).enumerate() {
-        println!("#{}: {:#?}", i, section);
-    }
+    //for (i, section) in anlz.sections(input).enumerate() {
+    //    println!("#{}: {:#?}", i, section);
+    //}
```

Comment:
Why did you comment this out?

Reply:
Ah, it's obsolete because all sections are parsed upfront now.

Reply:
I'll remove it.

Reply:
So they are parsed as part of the FileHeader?

Reply:
Not really. Basically the file is structured in sections consisting of Header and Content. There is exactly 1 outer section, and the content part contains the other sections. I introduced this separation and only parsed the header of the outer section because I thought this would be more memory efficient. Considering that these files are 5 - 100KiB for the demo tracks, I don't think thats a big issue though.